### PR TITLE
Feature/ad 16 user profile image

### PR DESCRIPTION
**Pull request by AlexW555**
The changes in this PR introduces the ability for users to upload their own profile pictures

- Assigns users' default avatars when they sign up from a random list of 20 local avatars
- Introduced field in edit profile for users to upload their own avatars from local file storage
- Modified seed to assign profile pictures to existing users

**A User changing their profile picture**

https://user-images.githubusercontent.com/17786854/130073414-01b98855-df8e-4de5-9378-422371a67207.mov

**A New user being assigned their profile picture**

https://user-images.githubusercontent.com/17786854/130073446-d9e48db2-df47-4137-946e-ba17f0594221.mov

### CODE REVIEW

**Comment by IeuanSkinner** [comment]
@AlexW555 have you created a ticket to address ActiveStorage ensuring we remove images when a user either changes their profile picture or deletes their profile?

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <section class="form container">
   <h2>Account settings</h2>
   <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |form| %>
+    <div class="field">
+      <%= image_tag url_for(resource.avatar), class: 'user-icon', alt: 'avatar' %>
+      <%= form.file_field :avatar %>
```
Comment (IeuanSkinner) [inline app/views/devise/registrations/edit.html.erb:6]: Is this an unrestricted upload? We should [limit](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept) these to image types, ideally just JPG and PNG with a size limit too.

**Comment by AlexW555** [comment]
> ActiveStorage ensuring we remove images when a user either changes their profile picture or deletes their profile?

Created that now [AD-43](https://lilw.atlassian.net/browse/AD-43?atlOrigin=eyJpIjoiNjE2MmUzYTQyMzA0NGJjYTgwNjFmOWI0ZDdkZmNiYjAiLCJwIjoiaiJ9)

File: app/views/devise/registrations/edit.html.erb (hunk 1 above)
Comment (AlexW555) [reply app/views/devise/registrations/edit.html.erb]: 100%, Added this in 👍🏻

**Comment by IeuanSkinner** [comment]
@garethdandrews / @phil-6 can you each review this branch please?

**Review by garethdandrews — COMMENTED** [review]
LGTM

**Pull request merged**